modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs

```diff
@@ -445,6 +445,28 @@ public static real_t MoveToward(real_t from, real_t to, real_t delta)
             return from + (Sign(to - from) * delta);
         }
 
+        /// <summary>
+        /// Moves <paramref name="from"/> toward <paramref name="to"/> by the <paramref name="delta"/> angle (in radians).
```

Comment:
This line looks a bit long, we can split it up:

```suggestion
        /// Moves <paramref name="from"/> toward <paramref name="to"/>
        /// by the <paramref name="delta"/> angle (in radians).
```